fee_allocator/accounting/fee_pipeline.py

```diff
@@ -64,6 +64,9 @@ def run_fees(
         pools = core_pools.get(chain.value, None)
         if pools is None:
             continue
+        if chain.value == Chains.ZKEVM.value:
```

Comment:
Not a huge fan of hard-coded fixes but you have another PR ready so that's ok for now :)